assets/js/components/notifications/FirstPartyModeSetupSuccessSubtleNotification.stories.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * AudienceSegmentationSetupSuccessSubtleNotification Component Stories.
+ *
+ * Site Kit by Google, Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import WithRegistrySetup from '../../../../tests/js/WithRegistrySetup';
+import { CORE_SITE } from '../../googlesitekit/datastore/site/constants';
+import { withNotificationComponentProps } from '../../googlesitekit/notifications/util/component-props';
+import FirstPartyModeSetupSuccessSubtleNotification from './FirstPartyModeSetupSuccessSubtleNotification';
+
+const NotificationWithComponentProps = withNotificationComponentProps(
+	'setup-success-notification-fpm'
+)( FirstPartyModeSetupSuccessSubtleNotification );
+
+function Template() {
+	return <NotificationWithComponentProps />;
+}
+
+export const Default = Template.bind( {} );
+Default.storyName = 'FirstPartyModeSetupSuccessSubtleNotification';
+Default.scenario = {};
+
+export default {
+	title: 'Modules/FirstPartyMode/Dashboard/FirstPartyModeSetupSuccessSubtleNotification',
+	decorators: [
+		( Story ) => {
+			const setupRegistry = ( registry ) => {
+				registry
+					.dispatch( CORE_SITE )
+					.receiveGetFirstPartyModeSettings( {
+						isEnabled: true,
+					} );
+			};
+
+			return (
+				<WithRegistrySetup func={ setupRegistry }>
+					<Story />
+				</WithRegistrySetup>
+			);
+		},
+	],
```

Comment:
I think we can get away with not mutating the registry here, as the component should render regardless. What do you think?

```suggestion
```

Reply:
You're quite right, this is unnecessary and I've removed it. Thanks for the spot!